src/azure-cli/HISTORY.rst

```diff
@@ -3,16 +3,64 @@
 Release History
 ===============
 
+**AMS**
+* Update show commands to return 3 if a resource is not found.
+
+**Azure Red Hat OpenShift**
```

Comment:
Why do you add so many release notes? Can you rebase dev branch and rearrange your release note?

Reply:
Apologies for this file. I think I pulled a version from the main fork rather than my fork.
I reverted this commit and pushed a new change.